### DISCUSSION TITLE
Error when running cs tools

### DISCUSSION THREAD

**Discussion by ThaoTranHexagon** [thoughtspot/cs_tools#133] [General]
When I ran this command, I got he below error
'.\cs_tools.exe tools git branches commit --tag myTag --branch-name myBranch --comment "myComment" --delete-aware'

'debug': {
                'code': 15012,
                'incident_id_guid': 'b3c13d82-854c-4de1-850c-839529cc3ead',
                'debug': 'Repository not in the correct state for the 
operation.Exception:Cannot pull into a repository with state: REBASING_MERGE'
            }

**Comment by boonhapus** [comment]
Hey @ThaoTranHexagon -- this looks like a generic ThoughtSpot git/vcs issue instead of a CS Tools one. Can you open a support ticket and share this one there?

**Reply by boonhapus** [reply]
Specifically -- this will be with the commit API. If you share your CS Tools log file here, I can highlight which exact API call was the issue.

`cs_tools logs report --latest 5`
